crates/server/openapi.yaml

```diff
@@ -802,17 +805,22 @@ components:
               maxLength: 1048576
               example: "ZXhhbXBsZQ=="
         - $ref: "#/components/schemas/Signature"
-    PublishedEnvelopeBody:
+    PublishedRecordEnvelope:
       description: A signed envelope body with the published registry log index.
       allOf:
         - type: object
           required:
             - registryIndex
+            - fetchToken
           properties:
             registryIndex:
               type: integer
               description: The index of the published record in the registry log.
               example: 42
+            fetchToken:
+              type: string
+              description: The fetch token for the registry log. Used for pagination in the fetch logs endpoint.
```

Comment:
:shrug: 
```suggestion
              description: The fetch token for the registry log. Used for incremental sync in the fetch logs endpoint.
```

Reply:
`Used as a cursor for incrementally fetching log records.` ?